third_party/packages/ansi_up/lib/ansi_up.dart

```diff
@@ -6,18 +6,17 @@
 /// codes.
 
 class AnsiUp {
-  AnsiUp() {
+  AnsiUp() : bold = false {
     _setupPalettes();
-    bold = false;
   }
 
-  String _text;
+  late String _text;
   bool bold;
-  List<List<AnsiUpColor>> ansiColors;
-  List<AnsiUpColor> palette256;
-  AnsiUpColor fg;
-  AnsiUpColor bg;
-  RegExp _csiRegex;
+  late List<List<AnsiUpColor>> ansiColors;
```

Comment:
try to avoid `late` when it is not needed as it has some performance overhead and gives you less static checking than other non-nullable types. In this case, _setupPlettes can just be inlined in the constructor to void the late.

Reply:
done